lib/generators/solid_cache/install/install_generator.rb

```diff
@@ -16,9 +16,65 @@ def create_config_solid_cache_yml
     template "config/solid_cache.yml"
   end
 
+  def add_cache_db_to_database_yml
+    if app_root.join("config/database.yml").exist?
+      if app_is_using_sqlite?
+        gsub_file database_yml, /production:\s*<<: \*default.*/m, sqlite_database_config_with_cache
```

Comment:
@dhh what you think if we removed nested if statements?

```
def add_cache_db_to_database_yml
  return unless app_root.join("config/database.yml").exist?

  if app_is_using_sqlite?
    update_database_yml_with_cache(sqlite_database_config_with_cache)
  else
    update_database_yml_with_cache(generic_database_config_with_cache)
  end
end

private

def update_database_yml_with_cache(config)
  gsub_file database_yml, /production:\s*<<: \*default.*/m, config
end
```

Reply:
I actually want to get rid of these defensive clauses entirely. They're only used for testing. It's fine for the installer to explode if you don't have config/database.yml.